azext_iot/_help.py

```diff
@@ -912,3 +912,253 @@
+    - name: List all IoT Plug and Play interfaces on a device.
+      text: >
+        az iot dt list-interfaces --login {iothub_cs}
+        --device-id {device_id}
+    - name: List all IoT Plug and Play interfaces on a device within current session.
+      text: >
+        az iot dt list-interfaces --hub-name {iothub_name} --device-id {device_id}
+"""
+
+helps['iot dt list-properties'] = """
+    type: command
+    short-summary: List properties of a target IoT Plug and Play device interface(s).
+    long-summary: You can leverage az login and provide --hub-name instead of --login for every command.
+    examples:
+    - name: List all properties of all interfaces on an IoT Plug and Play device.
+      text: >
+        az iot dt list-properties --login {iothub_cs}
+        --device-id {device_id}
+    - name: List all properties of all interfaces published on an IoT Plug and Play device within current session.
+      text: >
+        az iot dt list-properties --hub-name {iothub_name} --device-id {device_id}
+    - name: List all properties of a interface on an IoT Plug and Play device.
+      text: >
+        az iot dt list-properties --login {iothub_cs}
+        --device-id {device_id} --interface {plug_and_play_interface}
+"""
+
+helps['iot dt list-commands'] = """
+    type: command
+    short-summary: List commands of an IoT Plug and Play devices interface(s).
+    long-summary: You can leverage az login and provide --hub-name instead of --login for every command.
+    examples:
+    - name: List all commands of all interfaces on an IoT Plug and Play device.
+      text: >
+        az iot dt list-commands --login {iothub_cs}
+        --device-id {device_id} --repo-id {plug_and_play_model_repository_id}
+    - name: List all commands of an interface on an IoT Plug and Play device.
+      text: >
+        az iot dt list-commands --login {iothub_cs}
+        --device-id {device_id} --repo-id {plug_and_play_model_repository_id}
+        --interface {plug_and_play_interface}
+    - name: List all commands of all public interfaces on an IoT Plug and Play device.
+      text: >
+        az iot dt list-commands --login {iothub_cs}
+        --device-id {device_id}
+    - name: List all commands of public interface on an IoT Plug and Play device.
+      text: >
+        az iot dt list-commands --login {iothub_cs}
+        --device-id {device_id} --interface {plug_and_play_interface}
+"""
+
+helps['iot dt monitor-events'] = """
+    type: command
+    short-summary: Monitor Plug & Play device interface events.
+    long-summary: You can leverage az login and provide --hub-name instead of --login for every command.
+    examples:
+    - name: Monitor an IoT Plug and Play device events.
+      text: >
+        az iot dt monitor-events --login {iothub_cs} --device-id {device_id}
+        --interface {plug_and_play_interface} --consumer-group {consumer_group_name}
+    - name: Monitor an IoT Plug and Play device events within current session.
```

Comment:
The monitoring descriptions + examples should probably replace `IoT Plug and Play` to  `Digital Twin` events. Especially when `dt` is in the command.

Reply:
changed to digitaltwin